### PR TITLE
Risk image

### PR DESCRIPTION
**Pull request by ale-rt**
This is based on PR #204 to run the upgrade steps to create the missing columns.
You may decide to merge this one and ignore #204

There are some weird injection problems that cause unpredicted behavior, see the related ticket https://github.com/syslabcom/oira.prototype/issues/141

### CODE REVIEW

File: versions.cfg [hunk 1]
```diff
@@ -10,7 +10,9 @@ z3c.recipe.scripts = 1.0.1
 plone.recipe.zope2instance  = 4.4.0
 
 # Euphorie dependencies
+alembic = 1.1.0
```
Comment (pysailor) [inline versions.cfg]: Do we need to add 
`python-editor = 1.0.4`
as well? It is a dependency of alembic.

Comment (ale-rt) [reply versions.cfg]: I missed it, thanks for noting it, I added this in d978ce2

File: src/euphorie/deployment/upgrade/alembic/versions/24_.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        "risk", sa.Column("image_data_scaled", sa.LargeBinary(), nullable=True)
+    )
+    op.add_column("risk", sa.Column("image_filename", sa.UnicodeText(), nullable=True))
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column("risk", "image_data")
+    op.drop_column("risk", "image_filename")
```
Comment (pysailor) [inline src/euphorie/deployment/upgrade/alembic/versions/24_.py]: Just a doubt (not important): the downgrade should probable also drop column `image_data_scaled`?

Comment (ale-rt) [reply src/euphorie/deployment/upgrade/alembic/versions/24_.py]: I think we will never used downgrades but anyway I pushed the requested change

**Pull request merged**